ani-cli

```diff
@@ -444,8 +435,8 @@ case "$player_function" in
     download) dep_ch "ffmpeg" "aria2c" ;;
     android*) printf "\33[2K\rChecking of players on Android is disabled\n" ;;
     *iSH*) printf "\33[2K\rChecking of players on iOS is disabled\n" ;;
-    *iina*) true ;; # handled out of band in where_iina
-    *mpv*) true ;; # handled out of band in where_mpv
+    *IINA*) true ;; # handled out of band in where_iina
+    flatpak_mpv) true ;; # handled out of band in where_mpv
```

Comment:
Why the change to uppercase iina?

Reply:
this is to only match for /Applications/IINA.app/Contents/MacOS/iina-cli since we already use `-e` in where_iina , `iina` will be dep_ch by the catchall * argument